### PR TITLE
[FEATURE] Add public index files and Bootstrap::dispatch

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

**Comment by oliverklee** [comment]
Let's see what Travis thinks. I might need to do some whitelisting.

**Comment by oliverklee** [comment]
The build is green now.

File: Classes/Core/Bootstrap.php [hunk 1]
```diff
@@ -11,7 +11,12 @@
 /**
  * This class bootstraps the phpList core system.
  *
- * Include it in from the entry point and call Bootstrap::getInstance()->configure().
+ * Include it in from the entry point and call Bootstrap::getInstance() to get an instance,
```
Comment (samtuke) [inline Classes/Core/Bootstrap.php]: Is this a typo? E.g. "Include it_from the"

Comment (oliverklee) [inline Classes/Core/Bootstrap.php]: oh

Comment (oliverklee) [inline Classes/Core/Bootstrap.php]: Good catch! Fixed.

**Pull request merged**